tests/test_class_attributes.rs

```diff
@@ -96,7 +96,7 @@ fn recursive_class_attributes() {
 }
 
 #[test]
-#[cfg(panic = "unwind")]
+#[cfg_attr(cfg_panic, cfg(panic = "unwind"))]
```

Comment:
I wonder if we actually need this? IIRC, libtest does not work without unwinding in any case?

Reply:
Apparently not as the cfg is name `panic_unwind` in `print_feature_cfgs` but `cfg_panic` here and still everything passed. Will remove this in a follow up.

Reply:
Haha oops 😅